src/button-groups/button-radio.ts

```diff
@@ -0,0 +1,41 @@
+import { Directive, Input, HostListener, ElementRef, Renderer } from 'angular2/core';
+import { Subscription } from 'rxjs/Subscription';
+import { NglButtonGroup } from './button-group';
+
+@Directive({
+  selector: '[nglButtonRadio]',
+})
+export class NglButtonRadio {
+  value: any;
+
+  constructor(public element: ElementRef, public renderer: Renderer, public group: NglButtonGroup) { }
+
+  @Input('nglButtonRadio') set _value(_value: any) {
+    this.value = _value;
+
+    this.unsubscribe();
+    this._subscription = this.group.values.subscribe((_value) => {
```

Comment:
Just a suggestion: if a button is going to be always initialized with something (i.e., the typical case / i.e., a value is always required), then you could move the subscription logic in `ngOnInit` and subscribe only once. In `ngOnInit`, any data bound property has been initialized according to angular's spec.